src/main/kotlin/demo/controllers/OrganizationController.kt

```diff
@@ -0,0 +1,49 @@
+package demo.controllers
+
+import demo.models.people.Organization
+import demo.services.OrganizationService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@RequestMapping("/api/organizations")
+class OrganizationController(@Autowired private val organizationService: OrganizationService) {
+    @GetMapping
+    fun getOrganizations(): ResponseEntity<MutableIterable<Organization>> {
+        return ResponseEntity(organizationService.getOrganizations(), HttpStatus.OK)
+    }
+
+    @GetMapping("/{organizationId}")
+    fun getOrganization(@PathVariable organizationId: String): ResponseEntity<Organization> {
+        val organization: Optional<Organization> = organizationService.getOrganization(organizationId)
```

Comment:
Ten Optional jest z repozytorium konieczny, czy można go zastąpić kotlinowym Organization? (z tym pytajnikiem na końcu)

Reply:
Interfejs CrudRepository ma domyślnie optionale jako zwracane typy w metodach, do których odwołuje sie serwis, można ew. opakować to w serwisie jeśli Optional bardzo kole w oczy xD

Reply:
Nie wiem, czy aż tak kole w oczy, ale jeśli korzystamy z Kotlina, to skorzystałbym z uroków Kotlina - jeden z moich ostatnich komentarzy jest o tym XD